lib/playground.dart

```diff
@@ -1002,20 +1004,30 @@ class Playground implements GistContainer, GistController {
   }
 
   void _showSharingPage() {
-    window.open('https://github.com/dart-lang/dart-pad/wiki/Sharing-Guide',
-        'DartPad Sharing Guide');
+    window.location.href =
+        'https://github.com/dart-lang/dart-pad/wiki/Sharing-Guide';
```

Comment:
Is the result of this change that these pages now open in the existing tab/window as opposed to a new one?

Reply:
Yeah, that's a good point. Beforehand we would open the page in a new tab.